CHANGELOG.md

```diff
@@ -1,16 +1,223 @@
+v1.5.0
+------
+* **Release date:** 2018-09-??.
+* **Requires:** [tileserver v2.2.0](https://github.com/mapzen/tileserver/releases/tag/v2.2.0) and [tilequeue v2.1.0](https://github.com/mapzen/tilequeue/releases/tag/v2.1.0)  and [rawr-tiles v1.0.0](https://github.com/mapzen/raw-tiles/releases/tag/v1.0.0) and [coanacatl v1.0.0](https://github.com/mapzen/coanacatl/releases/tag/v1.0.0).
+
+  #### ENHANCEMENTS
+
+  * **buildings**: add `entrance` points, with optional **kind_detail** property with values: `main`, `staircase`, `service`, `home`, `unisex`, `garage`, `residence`, `private`, or `secondary`.
+  * **buildings**: add `exit` points, with optional **kind_detail** property with values: `emergency` or `fire_exit`.
+  * **buildings**: all building polygons are now clipped to tile boundaries, which fixes missing **landuse_kind** values. [Issue #1226](https://github.com/tilezen/vector-datasource/issues/1226) and [#1142](https://github.com/tilezen/vector-datasource/issues/1142) and [#487](https://github.com/tilezen/vector-datasource/issues/487).
+  * **buildings**: Add **building_material** optional property to describe the material covering the outside of the building or building part. Common values are: `brick`, `cement_block`, `clay`, `concrete`, `glass`, `masonry`, `metal`, `mud`, `other`, `permanent`, `plaster`, `sandstone`, `semi-permanent`, `steel`, `stone`, `timber-framing`, `tin`, `traditional` and `wood`. [Issue #1408](https://github.com/tilezen/vector-datasource/issues/1408).
+  * **landuse**: Add new kind values (listed below) to support full compatibility with OSM.org [#1425](https://github.com/tilezen/vector-datasource/issues/1425) map style.
```

Comment:
Looks weird to me to have "OSM" capitalised and with the `.org` ending.

Reply:
Fixed in 9edd7b94ad052aec7058090ba10153c50f7e561b.